snippets/systemrdl.json

```diff
@@ -0,0 +1,124 @@
+{
+    "Addrmap": {
+        "prefix": ["addrmap"],
+        "body": ["addrmap ${1:blkname} {", 
+            "\tname = \"$2\";",
+            "\tdesc = \"$3\";",
+            "\tdefault regwidth = 32;",
+            "\tdefault sw = rw;",
+            "\tdefault hw = r;",
+            "\trsvdset = 0;", 
```

Comment:
Would prefer the addrmap snippet to not include as many properties at the outset as they are not as universally used.

Also, the addrmap snippet appears to be missing the closing curly brace `}`

Reply:
Good suggestions.  Fixed

Reply:
Alex,
Any feedback?